package.json

```diff
@@ -40,9 +40,9 @@
     "react-use": "^15.3.3"
   },
   "devDependencies": {
-    "@backstage/cli": "^0.1.1-alpha.24",
-    "@backstage/dev-utils": "^0.1.1-alpha.24",
-    "@backstage/test-utils": "^0.1.1-alpha.24",
+    "@backstage/cli": "^0.2.0",
+    "@backstage/dev-utils": "^0.1.2",
```

Comment:
Is this correct? Definitely not `@backstage/dev-utils` version `^0.2.0`?

Reply:
Yes it is correct. For example api-docs plugin package.json on backstage:
https://github.com/backstage/backstage/blob/master/plugins/api-docs/package.json